index.js

```diff
@@ -107,6 +107,18 @@ app.get('/shoes', async (req, res) => {
     }
 });
 
+app.post('/shoes', async (req, res) => {
+
+    const shoeData = {
+        shoeId: 100,
+        brand: "Nike",
+        model: "Air Max",
+        size: 9
+    };
+    const shoey = await redisClient.json.set(`shoe:100`, '.', shoeData);
+    res.json(shoey)
+});
+
```

Comment:
This looks like an old version, can you update it to have the same stuff from line 190 on this index.js: https://github.com/Ensign-College/boxes-api-LittleExxGames/pull/6/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346